app/views/layouts/unifiedLayout.scala.html

```diff
@@ -98,16 +98,13 @@
 
 @beforeContentBlock = {
     @phaseBanner("beta", isAgent)
-    @hmrcBanner(Banner(CurrentLanguage()))
```

Comment:
should fix: remove - 

`@this(
    appConfig: FrontendAppConfig,
   -- hmrcBanner: HmrcBanner, --`